examples/worlds/fuel.sdf

```diff
@@ -122,6 +122,7 @@
 
     <!-- Included model without meshes -->
     <include>
+      <name>Double pendulum</name>
       <pose>-3 0 0 0 0 0</pose>
       <uri>https://fuel.ignitionrobotics.org/1.0/nate/models/double_pendulum_with_base/2</uri>
```

Comment:
I think @nkoenig removed some of these models from his account? I tried running this `fuel.sdf` world but fuel client complains about failing to download `double_pendulum_with_base` and `my_Backpack`, and ign-gazebo then exits.

Reply:
Yeah those will be fixed when https://github.com/ignitionrobotics/ign-gazebo/pull/219 is merged forward.